### PR TITLE
Listen for clicks on the edit button. Render the edit form (fix bug).

### PR DESCRIPTION
**Pull request by jwq1**
This PR renders the edit form when the edit button is clicked. 

A gif of the feature is attached. Ugly, but functional.

Formatting to come in a later PR.

### CODE REVIEW

**Comment by jwq1** [comment]
pr60

File: final-project/static/scripts.js [hunk 1]
```diff
@@ -247,68 +253,88 @@ function editProduct() {
...
+    // Otherwise, on all other elements use the innerText as a placeholder.
+    } else {
 
       // Create input element.
-      inputElement = document.createElement('input');
+      var inputElement = document.createElement('input');
       // Name the element.
       inputElement.setAttribute('name', key + '-input');
+      // Provide inner text of current placeholder.
+      inputElement.setAttribute('placeholder', value.innerText)
```
Comment (hdavidzhu) [inline final-project/static/scripts.js]: (nit) It's good to be consistent with using or not using semi-colons.

**Review by hdavidzhu — APPROVED** [review]

File: final-project/static/scripts.js (hunk 1 above)
Comment (jwq1) [reply final-project/static/scripts.js]: Definitely. Good catch. I think I can use JSLint to scan the whole file for stuff like this. Will do.

Comment (jwq1) [reply final-project/static/scripts.js]: This is completed in the https://github.com/jwq1/CS50x_jwq1/pull/60/commits/7cca3cddd72cb4cf4ca2c0a09d8fd2a7e0dc249b commit below

**Pull request merged**